services/structureddate/structureddate/src/main/java/org/collectionspace/services/structureddate/antlr/ANTLRStructuredDateEvaluator.java

```diff
@@ -220,6 +221,26 @@ public void exitUncertainDate(UncertainDateContext ctx) {
 		int earliestInterval = DateUtils.getCircaIntervalYears(earliestDate.getYear(), earliestDate.getEra());
 		int latestInterval = DateUtils.getCircaIntervalYears(latestDate.getYear(), latestDate.getEra());
 
+		if (earliestDate.getEra() == Era.BCE && latestDate.getEra() == Era.BCE) {
+			// Improved precision for BC dates
+			if ((latestDate.getMonth() == 12 && latestDate.getDay() == 31) &&
+				(earliestDate.getMonth() == 1 && earliestDate.getDay() == 1)) {
```

Comment:
Are there existing BC dates that need to be recomputed? If so, please make a JIRA.

Do other deployments besides PAHMA have BC dates? If so, ask if this change is ok for them.

Reply:
Only pahma requested this.  The JIRA is ay HM-24

Reply:
Right, I'm trying to avoid having a structured date parser that behaves differently for each museum. That would become a nightmare to maintain.